src/ExportHistory/Client/ExportHistoryClient.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask.ExportHistory;
+
+/// <summary>
+/// Convenience client for managing export jobs via entity signals and reads.
+/// </summary>
+public abstract class ExportHistoryClient
```

Comment:
What's the reasoning for having the `ExportHistoryClient` only do job operations but there is also a `ExportHistoryJobClient` that does other job operations?